API.md

```diff
@@ -0,0 +1,57 @@
+# Octofet API
+
+## `class Octofet`
+
+Uses: class Spi2Parallel
```

Comment:
What does it mean? To be more precise, why should it be interesting to a documentation reader?

Reply:
We say to the user: Spi2Parallel.cpp/h must be present in the src catalog.
Spi2Parallel is a low-level library for all pseudo-SPI devices. It supports both hardware and software SPI and provides data exchange between the controller and the chain. It is a common part for Octofet, MobyDig(big 7-segment), and we can use it with QuadDisplay2 and AmperkaFet.

Reply:
I don’t get it. I bought Octofet, I see the phrase «Uses: class Spi2Parallel»… So what? What should I do? Will it work out of the box?

- If yes, remove this phrase
- If no, tell me (as to a user) what and why should I do to get it working